randomstringutils.go

```diff
@@ -101,7 +102,21 @@ Returns:
 	error - an error stemming from an invalid parameter within underlying function, RandomSeed(...)
 */
 func RandomAlphaNumeric(count int) (string, error) {
-	return Random(count, 0, 0, true, true)
+	RandomString, err := Random(count, 0, 0, true, true)
+	if err != nil {
+		return "", fmt.Errorf("Error: %s", err)
+	}
+	match, _ := regexp.MatchString("([0-9]+)", RandomString)
```

Comment:
The only thing that catches my attention about this approach is the use of _ to avoid dealing with the error. The error is returned if the regex doesn't compile which shouldn't be an issue. But, gometalinter (and obviously one of the linters it uses) notes the issue.

I don't know that I mind this approach since I would be shocked if this regex didn't compile somewhere. This is more a note for myself if I ever get this setup working through a linter.